### PR TITLE
Release notes for Percona XtraBackup 2.3.4

### PR DESCRIPTION
**Pull request by hrvojem**

### CODE REVIEW

File: storage/innobase/xtrabackup/doc/source/release-notes/2.3/2.3.4.rst [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+============================
+
+Percona is glad to announce the release of |Percona XtraBackup| 2.3.4 on March 17th 2015. Downloads are available from our download site `here <http://www.percona.com/downloads/XtraBackup/xtrabackup-2.3.4/>`_. Downloads are available from our `download site <http://www.percona.com/downloads/XtraBackup/LATEST/>`_ and from :ref:`apt <apt_repo>` and :ref:`yum <yum_repo>` repositories.
+
+This release is the current GA (Generally Available) stable release in the 2.3 series.
+
+Bugs fixed
+----------
+
+ |Percona XtraBackup| didn't respect :option:`innodb_log_file_size` variable stored in :file:`backup-my.cnf`. Bug fixed :bug:`1527081`.
```
Comment (gl-sergei) [inline storage/innobase/xtrabackup/doc/source/release-notes/2.3/2.3.4.rst]: Lets say "didn't respect variables stored in 'backup-my.cnf' unless it was specified with --defaults-file

File: storage/innobase/xtrabackup/doc/source/release-notes/2.3/2.3.4.rst [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+ If server would run out of space while backups were taken with :option:`innobackupex --rsync` option backup process would fail but |innobackupex| would still complete with ``completed OK!`` message. Bug fixed :bug:`1537256`.
```
Comment (gl-sergei) [inline storage/innobase/xtrabackup/doc/source/release-notes/2.3/2.3.4.rst]: Lets say 'xtrabackup didn't abort backup if rsync completed with error'

File: storage/innobase/xtrabackup/doc/source/release-notes/2.3/2.3.4.rst [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+
+ |Percona XtraBackup| would not log master coordinates while performing backup on *MariaDB* 10. Bug fixed :bug:`1404484`.
```
Comment (gl-sergei) [inline storage/innobase/xtrabackup/doc/source/release-notes/2.3/2.3.4.rst]: didn't store gtid_slave_pos in xtrabackup_slave_info for mariadb 10 with GTID. It was only logged to stderr

**Comment by hrvojem** [comment]
Fixed as suggested

File: storage/innobase/xtrabackup/doc/source/release-notes/2.3/2.3.4.rst [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+============================
+
+Percona is glad to announce the release of |Percona XtraBackup| 2.3.4 on March 17th 2015. Downloads are available from our download site `here <http://www.percona.com/downloads/XtraBackup/xtrabackup-2.3.4/>`_. Downloads are available from our `download site <http://www.percona.com/downloads/XtraBackup/LATEST/>`_ and from :ref:`apt <apt_repo>` and :ref:`yum <yum_repo>` repositories.
+
+This release is the current GA (Generally Available) stable release in the 2.3 series.
+
+Bugs fixed
+----------
+
+ |Percona XtraBackup| didn't respect variables stored in :file:`backup-my.cnf` unless they were specified with :option:`xtrabackup --defaults-file` option. Bug fixed :bug:`1527081`.
```
Comment (gl-sergei) [inline storage/innobase/xtrabackup/doc/source/release-notes/2.3/2.3.4.rst]: unless they were specified -> unless it was specified. One needed to specify --defaults-file=backup-my.cnf for options to be respected.

**Comment by gl-sergei** [comment]
ok to merge with last comment addressed

**Pull request merged**